### ISSUE TITLE
How can we deploy hlf-kube on rancher

### ISSUE THREAD

**Issue by hsoltani** [hyfen-nl/PIVT#90]
I'm trying to deploy hlf-kube on rancher but I wonder how to implement init.sh command (./init.sh ./samples/simple/ ./samples/chaincode/) before running pods. How can I add crypto-config files and secrets to rancher?

**Comment by raftAtGit** [comment]
I dont think you need to do anything specific for Rancher. just setup your Kubernetes cluster with Rancher, and everything should work out of the box.

**Comment by hsoltani** [comment]
I'm trying to deploy hlf-kube as a new catalog in rancher and then launching the helm chart from the apps section so It's independent from the kubernetes cluster. Is it possible to write init.sh as a config file in hlf-kube helm chart? I'm launching hlf-kube in rancher but in my workload, the pods are down because config files and private keys (such as cert.pem) couldn't find.

**Comment by raftAtGit** [comment]
dont you have direct access to Kubernetes cluster? these Helm charts are meant to be used with CLI.

`init.sh` script does not interact with K8S cluster, it just creates necessary stuff and copies them to `hlf-kube` folder. you can possibly still use the populated `hlf-kube` folder in Rancher UI, you need to also somehow provide the values files to helm chart `samples/simple/network.yaml`, `samples/simple/crypto-config.yaml`.

later you need to run Argo flows to populate the Fabric network, which can be more tricky
